inst/rstudio/templates/project/RcppParallel.package.skeleton.dcf

```diff
@@ -0,0 +1,10 @@
+Binding: RcppParallel.package.skeleton
+Title: R Package using RcppParallel
+Subtitle: Create a new R Package using RcppParallel
+Caption: Create R Package using RcppParallel
+OpenFiles: Read-and-delete-me, src/vector-sum.cpp
```

Comment:
So what?  `READ-and-delete-me` is still stooopid, whether it is generated by R Core or not.  If we all agree that it is stooopid, why remove it?

(And if memory serves pkgKitten already does just that.)